packages/main/src/Calendar.ts

```diff
@@ -319,6 +330,32 @@ class Calendar extends CalendarPart {
 		this.secondaryCalendarType && this._setSecondaryCalendarTypeButtonText();
 	}
 
+	extractDateModes():string[] {
+		const format = this.getFormat() as DateFormat & { aFormatArray: Array<{type: string}> };
+		const types = format.aFormatArray.map((settings:{type:string}) => {
+			return settings.type.toLowerCase();
+		});
+		const result = [];
+		if (types.includes("day")) {
+			result.push("day");
+		}
+		if (types.includes("month") || types.includes("monthstandalone")) {
```

Comment:
The second part is redundant, this should also be fixed in OpenUI5. It's not possible to include the second part, but not the first.